client/templates/study_groups/edit_study_group_availability_slot_modal.html

```diff
@@ -0,0 +1,128 @@
+<template name="editStudyGroupAvailabilitySlotModal">
```

Comment:
maybe we could find a datepicker component for this :)

Reply:
done 👍